### PR TITLE
Expose dumping vfExec state

### PR DESCRIPTION
**Pull request by afk11**
Allows dumping the state of vfExec

### CODE REVIEW

File: btcdeb.cpp [hunk 1]
```diff
@@ -354,6 +356,7 @@ int main(int argc, char* const* argv)
         kerl_register("rewind", fn_rewind, "Go back in time one instruction.");
         kerl_register("stack", fn_stack, "Print stack content.");
         kerl_register("altstack", fn_altstack, "Print altstack content.");
+        kerl_register("vfexec", fn_vfexec, "Print altstack content.");
```
Comment (kallewoof) [inline btcdeb.cpp]: Description is wrong.

Comment (afk11) [reply btcdeb.cpp]: Thanks, fixed

**Comment by afk11** [comment]
Fixed nit, and a improper indentation in print_bool_stack

**Comment by kallewoof** [comment]
Thanks!

**Comment by sabran125** [comment]
5422cc

**Comment by sabran125** [comment]
54225cc

**Pull request merged**